src/System.Private.ServiceModel/src/System.Private.ServiceModel.csproj

```diff
@@ -1,8 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
-  <PropertyGroup>
```

Comment:
By removing this you are blocking anyone from building this project by itself as without it the configuration will default to "empty" and will either not build successfully or build something different then the default.